### PR TITLE
Handled `sent_to_provider_at` being nil

### PR DESCRIPTION
**Pull request by dcyoung-dev**
## Context

This should only be happening in the Support Console
Raises Sentry error https://dfe-teacher-services.sentry.io/issues/5678398911/?referrer=slack&alert_rule_id=853774&alert_type=issue

## Changes proposed in this pull request

- Added safety operator to handle nil values

## Guidance to review

N/A

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)
- [ ] Inform data insights team due to database changes
- [ ] Make sure all information from the Trello card is in here
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Add PR link to Trello card

### CODE REVIEW

**Comment by dcyoung-dev** [comment]
[Sent to provider at can be nil](https://trello.com/c/EcAAEXyT/2138-sent-to-provider-at-can-be-nil)

File: app/presenters/vendor_api/application_presenter/add_sent_to_provider_datetime.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module VendorAPI::ApplicationPresenter::AddSentToProviderDatetime
   def schema
     super.deep_merge!({
       attributes: {
-        sent_to_provider_at: application_choice.sent_to_provider_at.iso8601,
+        sent_to_provider_at: application_choice.sent_to_provider_at&.iso8601,
```
Comment (tomas-stefano) [inline app/presenters/vendor_api/application_presenter/add_sent_to_provider_datetime.rb:5]: I don't think this is needed 🤔, we should always return submitted apps in Vendor API. Is there any on our DB?

Comment (dcyoung-dev) [reply app/presenters/vendor_api/application_presenter/add_sent_to_provider_datetime.rb:5]: I fully agree but we've somehow triggered a Sentry error 🤷‍♂️

Comment (tomas-stefano) [reply app/presenters/vendor_api/application_presenter/add_sent_to_provider_datetime.rb:5]: I see what happened. We have the row in Support interface to see how the JSON Vendor API would be returned for that application choice. It makes sense.

**Review by tomas-stefano — APPROVED** [review]

**Pull request merged**